src/pybind/mgr/cephadm/ssl_cert_utils.py

```diff
@@ -121,3 +155,95 @@ def load_root_credentials(self, cert: str, priv_key: str) -> None:
         self.root_cert = given_cert
         self.root_key = serialization.load_pem_private_key(
             data=priv_key.encode('utf-8'), backend=default_backend(), password=None)
+
+
+def verify_tls(crt, key):
```

Comment:
yeah, that's the main concern is that we want this to be usable by all the modules. The issue with putting it here is that we end up in one of two scenarios: 1) we now just have duplicate code, both in places cephadm could access if it wanted, which doesn't seem correct or 2) we remove the duplicate in the mgr_utils in which case only cephadm can use this. Neither of them feel very good, so I'd prefer these functions just stay somewhere where all the modules can access them. If you want to move them around a bit and have them at the mgr level but in their own file it makes sense, but you then have to worry about changing every module that currently imports them from mgr_utils to now import from the new location. If you're willing to take that up then I'd be fine with you doing that but otherwise I think we should just leave them where they are.

Reply:
@adk3798 yeah, my final goal is to have all these functions in their own file (at the mgr level) and have all the modules use this new location. As I don't want to mess other modules on this PR I'll remove them from the new file (`ssl_cert_utils.py`). In a separate PR (once this PR is merged) I'll move everything to the new `ssl_cert_utils`, move this file to the main mgr module and update all the modules using it.